app/components/autocomplete/emoji_suggestion/emoji_suggestion.tsx

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import Fuse from 'fuse.js';
+import React, {useCallback, useEffect, useMemo, useRef} from 'react';
+import {FlatList, Platform, Text, View} from 'react-native';
+
+import {searchCustomEmojis} from '@actions/remote/custom_emoji';
+import {handleReactionToLatestPost} from '@actions/remote/reactions';
+import Emoji from '@components/emoji';
+import TouchableWithFeedback from '@components/touchable_with_feedback';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import {EmojiIndicesByAlias, Emojis} from '@utils/emoji';
+import {compareEmojis, getEmojiByName, getSkin} from '@utils/emoji/helpers';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import type CustomEmojiModel from '@typings/database/models/servers/custom_emoji';
+
+const EMOJI_REGEX = /(^|\s|^\+|^-)(:([^:\s]*))$/i;
+const EMOJI_REGEX_WITHOUT_PREFIX = /\B(:([^:\s]*))$/i;
+const REACTION_REGEX = /^(\+|-):([^:\s]+)$/;
```

Comment:
```suggestion
const REACTION_REGEX = /^([+-]):([^:\s]+)$/;
```

Would the above yield the same result ?